docs/configuring.md

```diff
@@ -113,6 +113,7 @@ The suggested minimum values for Java applications are shown in the following ta
 |**IPCSHMNSEGS**    | 10         |
 |**IPCSEMNIDS**	    | 500        |
 |**IPCSEMNSEMS**	  | 1000       |
+|**SHRLIBRGNSIZE**	  | 67 108 864       |
```

Comment:
@joransiu are these minimum values still appropriate for Java 11?

Reply:
The 64MB specified here is the default on zOS, and I think a reasonable amount to recommend for Java users too.

From my system, `java -version` w/ Java 8 consumed: 46,137,344 based on `D OMVS,L`.  Java 11 consumed: 50,331,648  I think 64 MB is a reasonable minimum size, given most users run one version of system installed Java.